### PR TITLE
[Dubbo-Container] Fix Enhance the java doc of dubbo-container module

### LINKED ISSUE #2994: Enhance the java doc of dubbo-container module.
Enhance the java doc of dubbo-container module.

Related project: https://github.com/apache/incubator-dubbo/projects/2

### PR DESCRIPTION
**Pull request by dreamer-nitj**
Fixes #2994

## What is the purpose of the change
This code change is to enhance the java doc for dubbo-container module for which issue was created at https://github.com/apache/incubator-dubbo/issues/2994

XXXXX

## Brief changelog

XXXXX

## Verifying this change

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [x] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/incubator-dubbo/tree/master/dubbo-test).
- [x] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/incubator-dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: dubbo-container/dubbo-container-api/src/main/java/org/apache/dubbo/container/Main.java [hunk 1]
```diff
@@ -33,6 +33,8 @@
 
 /**
  * Main. (API, Static, ThreadSafe)
+ *
+ * This class is entry point loading containers.
```
Comment (kezhenxu94) [inline dubbo-container/dubbo-container-api/src/main/java/org/apache/dubbo/container/Main.java]: Is this saying better? "This class is an entry point to load containers."

Comment (beiwei30) [reply dubbo-container/dubbo-container-api/src/main/java/org/apache/dubbo/container/Main.java]: I think it's fine.

**Comment by beiwei30** [comment]
LGTM.

**Comment by ralf0131** [comment]
@dreamer-nitj 

Hi, I am the moderator of the dev@dubbo.incubator.apache.org , I noticed that you've sent an email to this mailing list. But I failed to accept it due to Chinese New year vacation. Sorry about that, but since your pull request has been merged, feel free to send email if you want to :)

**Comment by dreamer-nitj** [comment]
Sure, thanks for the heads up :)

**Pull request merged**